src/Tag/Script.php

```diff
@@ -4,14 +4,20 @@
 
 namespace Yiisoft\Html\Tag;
 
+use Stringable;
 use Yiisoft\Html\Tag\Base\NormalTag;
 
 /**
  * @link https://www.w3.org/TR/html52/semantics-scripting.html#the-script-element
  */
 final class Script extends NormalTag
 {
+    public const NOSCRIPT_BEFORE = false;
+    public const NOSCRIPT_AFTER = true;
+
     private string $content = '';
+    private ?Noscript $noscript = null;
+    private bool $noscriptPosition = self::NOSCRIPT_AFTER;
```

Comment:
Does it make sense to add ability to customize poisition?

script-tag, noscript-tag and noscript-tag, script-tag will give exactly the same result since at the same time it can be interpreted either as noscript-tag or script-tag but not both.

I think it's better to remove position.

Reply:
Agree

Reply:
Done